CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # CHANGELOG
 
+## torchx-0.7.0
+
+* `torchx.schedulers`
+  * AWS Batch Scheduler
+    * Add job_role_arn and execution_role_arn as run options for AWS permission
+    * add instance type for aws_batch_scheduler multinode jobs
+    * Add neuron device mount for aws trn instances. 
+    * update EFA_DEVICE details for AWS resources
+  * Add aws_sagemaker_scheduler
```

Comment:
Are there more details in terms of feature parity for the new scheduler?

Reply:
from the message on the PR I didn't get that info. basically it's a scheduler to send job to aws segemaker.